Update docs for the notification-policy: update PR comment feature (#621)
spacelift-io/user-documentation: docs/concepts/policy/notification-policy.md

#### Updating an existing pull request comment

!!! info
    This feature is currently available for GitHub, GitHub Enterprise, GitLab, Bitbucket Cloud, and Azure DevOps VCS integrations.

>     We intend to add support for Bitbucket Data Center in the near future.

Reviewer: ```suggestion
    We intend to add support for Bitbucket Data Center soon.
```
